tests/test_urdf.py

```diff
@@ -0,0 +1,39 @@
+
+import math
+import os
+
+import pytest
+import xacro
+from urdf_parser_py.urdf import URDF
+
+model_data = [
+    ("med7", 25.5),
+    ("med14", 32.3),
+    ("iiwa7", 23.9),
+    ("iiwa14", 29.9)
+]
```

Comment:
Could you add a link to where these values come from.

Reply:
added links